openstack/utils/templates/_proxysql_configmap.yaml.tpl

```diff
@@ -0,0 +1,102 @@
+{{- define "proxysql_configmap" }}
```

Comment:
But it isn't using globals, it is getting the user credentials and databases of each database via the `.Values.mariadb`.

Reply:
Yes, for the simple case, where there is a single mariadb instance with a single user and schema, it could be done that way, but that doesn't cover ironic or nova.

Reply:
I See. The implementation is rather intrusive for services, but if proxysql is bringing the benefits you're promoting and due to the shortcomings of helm I assume there is no better solution than an operator. LGTM